services/notifier/base.go

```diff
@@ -58,3 +58,16 @@ func RunNotifier(database *db.Instance, delivery amqp.Delivery) error {
 
 	return nil
 }
+
+func UnprefixedAddress(address string) (string, uint, bool) {
```

Comment:
can you add test for this one just in case?

Reply:
done